internal/repository/implement/authentication_repository.go

```diff
@@ -47,12 +46,10 @@ func (repo *AuthenticationRepository) ValidateRefreshToken(ctx context.Context,
 	`
 	err := repo.db.GetContext(ctx, &authentication, query, customerId)
 	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, nil
+		}
```

Comment:
chỗ này sao sài sql.ErrNoRows vầy, mình sài sqlx mà. T có define 1 cái constants ở đây, xem thử nha

[https://github.com/21CLC01-WNC-Banking/WNC-Banking-BE/blob/56f19ee61d2798c0ad132bdca6e5979ddf08323f/internal/domain/http_common/constants.go#L36C1-L37C1](https://github.com/21CLC01-WNC-Banking/WNC-Banking-BE/blob/56f19ee61d2798c0ad132bdca6e5979ddf08323f/internal/domain/http_common/constants.go#L36C1-L37C1)

Reply:
Tính ra là thấy bây sài cái đó trước luôn, t chỉ sửa cái sai logic thôi

Reply:
này t ko bít nha